### PR TITLE
osd: remove duplicated "commit_queued_for_journal_write" in OpTracker

### PR DESCRIPTION
**Pull request by ashitakasam**
Fixes: http://tracker.ceph.com/issues/23440

Signed-off-by: ashitakasam <694240887@qq.com>

### CODE REVIEW

**Comment by ashitakasam** [comment]
```
                "events": [
                    {
                        "time": "2018-03-23 12:05:38.972791",
                        "event": "initiated"
                    },
                    {
                        "time": "2018-03-23 12:05:38.972791",
                        "event": "header_read"
                    },
                    {
                        "time": "2018-03-23 12:05:38.972795",
                        "event": "throttled"
                    },
                    {
                        "time": "2018-03-23 12:05:38.974553",
                        "event": "all_read"
                    },
                    {
                        "time": "2018-03-23 12:05:38.974621",
                        "event": "dispatched"
                    },
                    {
                        "time": "2018-03-23 12:05:38.974655",
                        "event": "queued_for_pg"
                    },
                    {
                        "time": "2018-03-23 12:05:38.974775",
                        "event": "reached_pg"
                    },
                    {
                        "time": "2018-03-23 12:05:38.975002",
                        "event": "started"
                    },
                    {
                        "time": "2018-03-23 12:05:38.975117",
                        "event": "waiting for subops from 0,2"
                    },
                    {
                        "time": "2018-03-23 12:05:45.467335",
                        "event": "commit_queued_for_journal_write"
                    },
                    {
                        "time": "2018-03-23 12:05:45.467340",
                        "event": "commit_queued_for_journal_write"
                    },
                    {
                        "time": "2018-03-23 12:05:45.482580",
                        "event": "write_thread_in_journal_buffer"
                    },
                    {
                        "time": "2018-03-23 12:05:45.861273",
                        "event": "sub_op_commit_rec from 0"
                    },
                    {
                        "time": "2018-03-23 12:05:46.071409",
                        "event": "sub_op_commit_rec from 2"
                    },
                    {
                        "time": "2018-03-23 12:05:46.099900",
                        "event": "journaled_completion_queued"
                    },
                    {
                        "time": "2018-03-23 12:05:46.100110",
                        "event": "op_commit"
                    },
                    {
                        "time": "2018-03-23 12:05:46.100163",
                        "event": "commit_sent"
                    },
                    {
                        "time": "2018-03-23 12:05:46.102404",
                        "event": "done"
                    }
                ]
```

**Comment by ashitakasam** [comment]
```
                "events": [
                    {
                        "time": "2018-03-23 13:25:08.628590",
                        "event": "initiated"
                    },
                    {
                        "time": "2018-03-23 13:25:08.628590",
                        "event": "header_read"
                    },
                    {
                        "time": "2018-03-23 13:25:08.628599",
                        "event": "throttled"
                    },
                    {
                        "time": "2018-03-23 13:25:08.632225",
                        "event": "all_read"
                    },
                    {
                        "time": "2018-03-23 13:25:08.632249",
                        "event": "dispatched"
                    },
                    {
                        "time": "2018-03-23 13:25:08.632263",
                        "event": "queued_for_pg"
                    },
                    {
                        "time": "2018-03-23 13:25:08.632330",
                        "event": "reached_pg"
                    },
                    {
                        "time": "2018-03-23 13:25:08.632352",
                        "event": "started"
                    },
                    {
                        "time": "2018-03-23 13:25:09.664333",
                        "event": "commit_queued_for_journal_write"
                    },
                    {
                        "time": "2018-03-23 13:25:13.188672",
                        "event": "write_thread_in_journal_buffer"
                    },
                    {
                        "time": "2018-03-23 13:25:13.509438",
                        "event": "journaled_completion_queued"
                    },
                    {
                        "time": "2018-03-23 13:25:13.515041",
                        "event": "commit_sent"
                    },
                    {
                        "time": "2018-03-23 13:25:13.515101",
                        "event": "done"
                    }
                ]
```

File: src/os/filestore/FileJournal.cc [hunk 1]
```diff
@@ -1609,8 +1609,6 @@ void FileJournal::submit_entry(uint64_t seq, bufferlist& e, uint32_t orig_len,
   assert(e.length() > 0);
   assert(e.length() < header.max_size);
 
-  if (osd_op)
-    osd_op->mark_event("commit_queued_for_journal_write");
```
Comment (Liuchang0812) [inline src/os/filestore/FileJournal.cc]: hi @tchaikov would you mind taking a look? should we delete this event directly or use other event name?  thank you

Comment (tchaikov) [reply src/os/filestore/FileJournal.cc]: i think we'd better remove this one. as their timestamp should always be identical.

**Comment by tchaikov** [comment]
@ashitakasam could you revise the title of your commit message so it describes your change instead of the bug symptom addressed by it?

**Comment by Liuchang0812** [comment]
@tchaikov ping

**Comment by tchaikov** [comment]
retest this please

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2018-04-12_15:56:07-rados-wip-kefu-testing-2018-04-12-2211-distro-basic-mira/

failed tests are 

- tracked by http://tracker.ceph.com/issues/23699
- caused by #19117, #21188
- fixed by #21406

**Pull request merged**